app/views/users/show.html.erb

```diff
@@ -1 +1,33 @@
-<h1>This is User details</h1>
\ No newline at end of file
+<div class="users_show_view">
+  <div class="user-container">
+    <div class="user-text">
+      <div class="user-name"><%= @user.name %></div>
+      <div class="user-posts">Posts: <%= @user.posts_counter %></div>
+      <div class="user-comments">Comments: <%= @user.comments_counter %></div>
+      <div class="user-likes">Likes: <%= @user.likes_counter %></div>
+    </div>
+    <div class="user-content">
+      <div class="user-text"><%= @user.text %></div>
+      <div class="user-date">Date: <%= @user.created_at %></div>
+    </div>
+
+    <div class="user-posts">
+      <div class="posts-container">
+        <% @user.posts.each do |post| %>
```

Comment:
* As shown in the wireframe you should render the 3 most recent posts as you have created the method already in your user model also ensure your views function as shown in the wireframe below 👇 
![image](https://user-images.githubusercontent.com/52497246/222758331-d7137f92-6657-4894-be44-4abb540e12e9.png)